lib/puppet/application/ssl.rb

```diff
@@ -13,41 +16,68 @@ def help
 
 SYNOPSIS
 --------
-Manage SSL keys and certificates for an SSL clients needed
+Manage SSL keys and certificates for SSL clients needing
 to communicate with a puppet infrastructure.
 
 USAGE
 -----
-puppet ssl <action> [--certname <NAME>]
+puppet ssl <action> [-h|--help] [-v|--verbose] [-d|--debug] [--localca]
+
+
+OPTIONS
+-------
+
+* --help:
+  Print this help messsge.
+
+* --verbose:
+  Print extra information.
+
+* --debug:
+  Enable full debugging.
+
+* --localca
+  Also clean the local CA certificate and CRL.
+
 
 ACTIONS
 -------
 
 * submit_request:
-  Generate a certificate signing request (CSR) and submit it to the CA. If a private and
-  public key pair already exist, they will be used to generate the CSR. Otherwise a new
-  key pair will be generated. If a CSR has already been submitted with the given `certname`,
-  then the operation will fail.
+  Generate a certificate signing request (CSR) and submit it to the CA. If
+  a private and public key pair already exist, they will be used to generate
+  the CSR. Otherwise a new key pair will be generated. If a CSR has already
+  been submitted with the given `certname`, then the operation will fail.
 
 * download_cert:
-  Download a certificate for this host. If the current private key matches the downloaded
-  certificate, then the certificate will be saved and used for subsequent requests. If
-  there is already an existing certificate, it will be overwritten.
+  Download a certificate for this host. If the current private key matches
+  the downloaded certificate, then the certificate will be saved and used
+  for subsequent requests. If there is already an existing certificate, it
+  will be overwritten.
 
 * verify:
-  Verify the private key and certificate are present and match, verify the certificate is
-  issued by a trusted CA, and check revocation status.
+  Verify the private key and certificate are present and match, verify the
+  certificate is issued by a trusted CA, and check revocation status.
+
+* clean:
+  Remove the private key and certificate related files for this host. If
+  `--localca` is specified, then also remove this host's local copy of the
+  CA certificate(s) and CRL bundle.
 HELP
   end
 
-  option('--certname NAME') do |arg|
-    options[:certname] = arg
```

Comment:
Looks like we can get rid of this because the option exists by default, since it's a puppet setting?

Reply:
Yep exactly. I forgot it was a setting when I added it, then realized it wasn't needed